components/search/MoreFiltersDropdownFacet.vue

```diff
@@ -1,24 +1,24 @@
 <template>
-  <b-dropdown-form>
```

Comment:
Each facet in the more filters dropdown is _not_ its own form, but should instead be a fieldset within the parent component's form.